accounting/create_db.py

```diff
@@ -10,50 +10,45 @@
 # SPDX-License-Identifier: LGPL-3.0
 ###############################################################
 import sqlite3
+import pandas as pd
+import logging
 
 
-def create_db():
+LOGGER = logging.basicConfig(filename="accounting/db_creation.log", level=logging.INFO)
+
+
+def create_db(filepath):
```

Comment:
This function should do some sanity check on the `filepath` input and tried to fail over if recoverable (e.g., missing subdirectories leading to the db file) or raise an exception or return an error code?